### ISSUE TITLE
bug: plex stuck on idle

### ISSUE THREAD

**Issue by Sav3k** [FoxxMD/multi-scrobbler#218]
### Please check existing knowledge before opening an issue

- [X] I have [checked the FAQ](https://foxxmd.github.io/multi-scrobbler/docs/FAQ/) and [documentation](https://foxxmd.github.io/multi-scrobbler/docs/configuration/)
- [X] I have [searched through existing issues](https://github.com/FoxxMD/multi-scrobbler/issues?q=sort%3Aupdated-desc) and [discussions.](https://github.com/FoxxMD/multi-scrobbler/discussions)

### Describe the Bug

After setting up Plex with the new API-method everything seemed to be working fine. When i proceeded to test the scrobling sometime later i noticed that plex was stuck on idle. The log shows an error there and i don't know if it's something with my setup or the app itself. 

this is what my config looks like: 
```
      {
        "name": "Plex",
        "enable": true,
        "clients": [],
        "data": {
          "token": "MYPLEXTOKEN",
          "url": "http://192.168.1.2:32400"
        },
        "type": "plex"
      }
```

### Platform

Docker

### Versions

- multi-scrobbler on v0.8.7

### Logs

```shell
[2024-10-30 14:35:43.393 +0100] ERROR  : [App] [Sources] [Plex - Plex] Error occurred while polling
[2024-10-30 14:35:43.393 +0100] ERROR  : [App] [Sources] [Plex - Plex] Cannot read properties of undefined (reading 'toLocaleLowerCase')
TypeError: Cannot read properties of undefined (reading 'toLocaleLowerCase')
    at PlexApiSource.isActivityValid (CWD/src/backend/sources/PlexApiSource.ts:215:88)
    at PlexApiSource.getRecentlyPlayed (CWD/src/backend/sources/PlexApiSource.ts:340:36)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at PlexApiSource.doPolling (CWD/src/backend/sources/AbstractSource.ts:389:34)
    at PlexApiSource.startPolling (CWD/src/backend/sources/AbstractSource.ts:324:27)
    at PlexApiSource.poll (CWD/src/backend/sources/AbstractSource.ts:292:9)
```

**Comment by FoxxMD** [comment]
Thanks for the report. To help me debug this can you please do the following:

* Use Plex with media/music playing the same way it was when this error occurred (or try your best to reproduce it)
* Visit this URL (use your own token) and copy/paste the response XML data that is rendered to a comment here

```
http://192.168.1.2:32400/status/sessions?X-Plex-Token=MYPLEXTOKEN
```

**Comment by FoxxMD** [comment]
Thanks for the quick feedback. Can you confirm for me whether you need to login to Plex or not? If you are on desktop does the profile area just show "Sign In" like this?

![image](https://github.com/user-attachments/assets/991de170-1981-4257-952f-6ec76e0184f4)

**Comment by Sav3k** [comment]
Thank you, for your quick help! :) It shows sign in yes, i did and the issue persists.

**Comment by Sav3k** [comment]
Okay, maybe it needs some time after the login because now it seems to be working fine again: 

```
[2024-10-30 17:06:32.210 +0100] INFO   : [App] [Sources] [Plex - Plex] Polling started
[2024-10-30 17:06:32.206 +0100] INFO   : [App] [Heartbeat] [Sources] Attempted to restart 1 sources that were not polling.
[2024-10-30 17:06:32.204 +0100] INFO   : [App] [Sources] [Plex - Plex] Should be polling! Attempting to restart polling...
```

**Comment by FoxxMD** [comment]
There is an issue! I have isolated it and am building a PR for it shortly for you to test.

**Comment by Sav3k** [comment]
Ah okay, good to know it's not me going mad

**Comment by FoxxMD** [comment]
@Sav3k please try using docker image `foxxmd/multi-scrobbler:pr-219` and let me know if that fixes the issue

**Comment by Sav3k** [comment]
Seems to be working, after signing out the log shows that it's still scrobbling and correctly using the `PLEX_LOCAL_USER`

**Comment by FoxxMD** [comment]
Fix is now available in `latest` tag or [0.8.8](https://github.com/FoxxMD/multi-scrobbler/releases/tag/0.8.8)

**Comment by Sav3k** [comment]
Thanks for the quick fix, appreciate it.
